src/utils.ts

```diff
@@ -188,15 +485,73 @@ export namespace Utils {
     transaction: Transaction,
     accountId: string,
   ): boolean {
+    return verifyTxMultiSignedBy(transaction, accountId).length !== 0;
+  }
+
+  /**
+   *
+   * verifyTxMultiSignedBy checks if a transaction has been signed by one or more of
+   * the signers, returning a list of signers that were found to have signed the
+   * transaction.
+   *
+   * @function
+   * @memberof Utils
+   * @param {Transaction} transaction the signed transaction.
+   * @param {string[]} accountIDs The signers public keys.
+   * @example
+   * let keypair1 = Keypair.random();
+   * let keypair2 = Keypair.random();
+   * const account = new StellarSdk.Account(keypair1.publicKey(), "-1");
+   *
+   * const transaction = new TransactionBuilder(account, { fee: 100 })
+   *    .setTimeout(30)
+   *    .build();
+   *
+   * transaction.sign(keypair1, keypair2)
+   * Utils.verifyTxMultiSignedBy(transaction, [keypair1.publicKey(), keypair2.publicKey()])
+   * @returns {string[]} a list of signers that were found to have signed the transaction.
+   */
+  export function verifyTxMultiSignedBy(
```

Comment:
maybe we should rename this function since the return value is not really verifying if the tx was multi signed by, but rather returning the list of accounts which were found as a signer to the tx.